README.md

```diff
@@ -130,24 +142,33 @@ https://docs.docker.com/engine/install/linux-postinstall/
 - For the simplest option, use MySQL Option 2 ("MySQL 5.6 and above in SDK docker container"), nothing further is required. 
 - (MG: I had a recent issue with Option 2 recently, but was able to get things to work following the Option 3 steps below) 
   
-- If you or need to connect to an existing OpenMRS database, use Option 3 in the SDK installation process. You will 
-need to create your own MySQL Docker container and instantiate a database into it:
-  
-  * Create a container (example below creates a container named "mysql-mirebalais" that will be available on port 3308):
+  - If you or need to connect to an existing OpenMRS database, or if you need a custom MySQL instance that has bin logging enabled, 
+    use Option 3 in the SDK installation process. You will need to create your own MySQL Docker container and instantiate a database into it:
   
-  ```shell script
-    docker run --name mysql-mirebalais -d -p 3308:3306 \
-            -e MYSQL_ROOT_PASSWORD=root \
-            mysql:5.6 --character-set-server=utf8 --collation-server=utf8_general_ci --max_allowed_packet=1G
-    ``` 
-  
-  * Get a bash shell in the container, and create an empty database to use
+    * Create a container (example below creates a container named "mysql-mirebalais" that will be available on port 3308):
   
     ```shell script
-       $ docker exec -it mysql-mirebalais bash
-       root@f25c851762df:/# mysql -uroot -proot
-       mysql> create database openmrs default charset utf8;
+      docker run --name mysql-mirebalais -d -p 3308:3306 \
+              -e MYSQL_ROOT_PASSWORD=root \
+              mysql:5.6 \
+              --character-set-server=utf8 \
+              --collation-server=utf8_general_ci \
+              --max_allowed_packet=1G \
+              --innodb-buffer-pool-size=2G \
+              --user=mysql \
+              --server-id=1 \
+              --log-bin=mysql-bin \
+              --binlog_format=ROW \
+              --max_binlog_size=100M
       ``` 
+  
+    * Get a bash shell in the container, and create an empty database to use
+  
+      ```shell script
+         $ docker exec -it mysql-mirebalais bash
+         root@f25c851762df:/# mysql -uroot -proot
+         mysql> create database openmrs default charset utf8;
```

Comment:
Not sure.  I always do it probably because I was burned with a character encoding issue in the past, but that shouldn't be an issue if the configuration for mysql is setup correctly.  In any event, I didn't add this - it was already in the README.

Reply:
Hmm... yeah, could be the character encoding...